Patches/MorrowRim/MorrowRim_DaedraCE.xml

```diff
@@ -0,0 +1,544 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+		<li>MorrowRim</li>
+		<li>MorrowRim - Just the Animals</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/ThingDef[defName="MorrowRim_Clannfear"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Quadruped</bodyShape>
```

Comment:
Bipedal, but its posture is similar to that of a raptor with ling arms, it is not an upright biped.

Reply:
Now that you mention it, it may actually be more suitable to use `Birdlike`. It's what chickens and patched raptors use, and it better fits their posture.

Reply:
They don't have long necks that raise heads above their shoulders like that of birds and raptors. I say quadruped fit them better.